lib/pf/radius.pm

```diff
@@ -392,6 +393,7 @@ sub accounting {
     my $timer = pf::StatsD::Timer->new();
     my ($self, $radius_request, $headers) = @_;
     my $logger = $self->logger;
+    local $pf::dal::node::TRIGGER_NODE_DISCOVERED = 1;
```

Comment:
I don't think it's necessary here because sending an accounting request for a node which doesn't exist will not create the node in PacketFence DB.